test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -2434,7 +2435,7 @@ TEST_P(LocalityMapTest, UpdateMap) {
   EXPECT_EQ(0U, backends_[3]->backend_service()->request_count());
   // Wait until the locality update has been processed, as signaled by backend 3
   // receiving a request.
-  WaitForBackend(3);
+  WaitForAllBackends(3, 4);
```

Comment:
What's the reason for this change?  It looks like the two expressions are equivalent.

Reply:
Ah, nevermind this question -- I didn't see your earlier comment until after I'd written this.